sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -557,12 +557,6 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.quirk_data = &adl_max98360a_amp,
 		.sof_tplg_filename = "sof-adl-max98360a-cs42l42.tplg",
 	},
-	/* place amp-only boards in the end of table */
-	{
-		.id = "CSC3541",
-		.drv_name = "adl_cs35l41",
-		.sof_tplg_filename = "sof-adl-cs35l41.tplg",
-	},
```

Comment:
moving this code block should be a separate patch.

Reply:
Thanks for your review. Moving part has been removed from this patch and will upload as separate commit.